packages/bpk-component-navigation-bar/README.md

```diff
@@ -33,7 +32,7 @@ export default () => (
       />
     }
     trailingButton={
-      <BpkButtonLink onClick={() => {}}>Done</BpkButtonLink>
+      <BpkNavigationBarButtonLink onClick={() => {}}>Done</BpkNavigationBarButtonLink>
```

Comment:
Maybe with the old branding this wasn't an issue, but given the default colour for `BpkButtonLink` is blue and the background colour of `BpkNavigationBar` is also blue, it's not possible to read the text in this case 😅